packages/widgets-manager/src/utils/firebase.ts

```diff
@@ -0,0 +1,94 @@
+// ts-ignore
+import * as Firebase from 'firebase/app';
+import 'firebase/auth';
+import 'firebase/firestore';
+import { updateUserData, syncData } from '../store/actions';
+
+const debug = require('debug')('widgets-manager:utils:firebase');
+
+const config = {
+  apiKey: "AIzaSyATST9016Y3SCCnyEM0quH1UoCfbv35MEo",
+  authDomain: "lt-generic-store.firebaseapp.com",
+  databaseURL: "https://lt-generic-store.firebaseio.com",
+  projectId: "lt-generic-store",
+  storageBucket: "lt-generic-store.appspot.com",
+  messagingSenderId: "855806900184",
+  appId: "1:855806900184:web:789d34f4cf40dbd1"
+}
```

Comment:
Should we store these things in a configuration file?

Reply:
I agree with Philipp that these values should be stored in some secret storage that permits us to change them without much blood and without touching the main code, also I think it should be flexible in env sence.